tests/testthat/test_save_plot.R

```diff
@@ -0,0 +1,42 @@
+library(ggplot2)
+library(tibble)
+
+context('Save Plots')
+
+# Dummy templates
+foo <- ggplot2::ggplot()
+bar <- ggplot2::ggplot()
+dummy_templates <- list('foo'=foo, 'bar'=bar)
+performer_id <- "baz"
+
+# Displaylab templates
+templates <- load_templates()
+
+test_that("templates are saved in save_path directory with .png format", {
+  save_path <- getwd()
+  save_plots(dummy_templates, save_path, performer_id)
+
+  current_files <- list.files(path=save_path, pattern=".png")
+  expected_files <- c("baz-bar.png", "baz-foo.png")
+  do_match <- current_files == expected_files
+  expect_true(all(do_match))
+  file.remove(current_files)
```

Comment:
This is the cleanup that will eventually go into an after block that runs regardless if the test errored out or not.